packages/shared/src/components/sidebar/sections/BookmarkSection.tsx

```diff
@@ -1,46 +1,85 @@
 import React, { ReactElement } from 'react';
 import { ListIcon, SidebarMenuItem } from '../common';
-import { BookmarkIcon, PlusIcon } from '../../icons';
+import { ArrowIcon, BookmarkIcon, PlusIcon } from '../../icons';
 import { Section } from '../Section';
 import { webappUrl } from '../../../lib/constants';
 import { SidebarSettingsFlags } from '../../../graphql/settings';
 import { SidebarSectionProps } from './common';
 import { useLazyModal } from '../../../hooks/useLazyModal';
 import { LazyModal } from '../../modals/common/types';
+import { BookmarkReminderIcon } from '../../icons/Bookmark/Reminder';
+import {
+  useBookmarkFolderList,
+  useCreateBookmarkFolder,
+} from '../../../hooks/bookmark';
+import { useViewSize, ViewSize } from '../../../hooks';
 
 export const BookmarkSection = ({
   isItemsButton,
   ...defaultRenderSectionProps
 }: SidebarSectionProps): ReactElement => {
   const { openModal } = useLazyModal();
+  const { folders } = useBookmarkFolderList();
+  const { createFolder } = useCreateBookmarkFolder();
+
+  const isLaptop = useViewSize(ViewSize.Laptop);
+  const rightIcon = !isLaptop && (() => <ArrowIcon className="rotate-90" />);
```

Comment:
This doesn't render in the sidebar though, not sure this adds anything here?
I think you want this for the inline mobile view right?

In that case we should actually abstract from sidebar and that view into re-usable component or simple share the list.
But maybe better if they just 2 different things?

Reply:
Yes, I can build a different component but reusing just the list is a mess because of all the sidebar items used inside that relies on sidebar props. Wdyt?

Reply:
Not sure I fully understand then what this is for?
(the !isLaptop) render

Reply:
Only on mobile/tablet we have the right icon, so i had to put a check before adding them 👀